File: DrinkItUpWebApp/DrinkItUpWebApp/Models/DrinkSearchModel.cs
using Microsoft.Identity.Client;

namespace DrinkItUpWebApp.Models
{
	public class DrinkSearchModel
	{
		public int DrinkId { get; set; }

		public string Name { get; set; }

		public MainAlcoholModel MainAlcohol { get; set; }

		public DifficultyModel Difficulty { get; set; }

		public string Description { get; set; }

		public Dictionary<string, List<DrinkSearchModel>> Results { get; set; }

        public string DrinkPhotoId { get; set; }
    }
}

File: DrinkItUpWebApp/DrinkItUpSPA/src/app/shared/Models/drinkSearch.model.ts
import { DifficultyModel } from "./difficulty.model";
import { MainAlcoholModel } from "./mainAlcohol.model";

export class DrinkSearchModel{
    constructor(
        public drinkId: number,
        public name: string,
        public difficulty: DifficultyModel,
        public mainAlcohol: MainAlcoholModel,
        public description: string,
        public drinkPhotoId: string
    ){}
}